Firestore/core/test/firebase/firestore/util/status_test_util.h

```diff
@@ -20,11 +20,31 @@
 #include "Firestore/core/src/firebase/firestore/util/status.h"
 #include "gtest/gtest.h"
 
+namespace firebase {
+namespace firestore {
+namespace util {
+
+inline testing::AssertionResult Equal(Status expected, Status actual) {
+  if (expected != actual) {
+    return testing::AssertionFailure()
+           << "Should have seen status " << expected.ToString() << " but got "
+           << actual.ToString();
+  }
+
+  return testing::AssertionSuccess();
+}
+
+}  // namespace util
+}  // namespace firestore
+}  // namespace firebase
+
 // Macros for testing the results of functions that return tensorflow::Status.
-#define EXPECT_OK(statement) \
-  EXPECT_EQ(::firebase::firestore::util::Status::OK(), (statement))
-#define ASSERT_OK(statement) \
-  ASSERT_EQ(::firebase::firestore::util::Status::OK(), (statement))
+#define EXPECT_OK(statement)                      \
```

Comment:
(line 41): Oops. I must've missed the "tensorflow" when I imported this. Feel free to leave as is, or to go ahead and fix it.

Reply:
Fixed this and a few more.